airflow/providers/google/cloud/operators/cloud_storage_transfer_service.py

```diff
@@ -885,7 +892,7 @@ def _create_body(self) -> dict:
             STATUS: GcpTransferJobsStatus.ENABLED,
             TRANSFER_SPEC: {
                 AWS_S3_DATA_SOURCE: {BUCKET_NAME: self.s3_bucket},
```

Comment:
Maybe we should add the option to include a source path here at the same time? It looks like the transferSpec supports a path argument for S3 bucket sources

https://cloud.google.com/storage-transfer/docs/reference/rest/v1/TransferSpec#AwsS3Data

Reply:
Thank you Sam, totally agree it'd bring great value 👍 and sorry I missed it